CMakeLists.txt

```diff
@@ -244,7 +259,7 @@ target_compile_definitions(kauai PUBLIC
 
 target_link_libraries(kauai
   PUBLIC
-    3DMMForever::AudioMan
+    "${AUDIOMAN_LIB}"
```

Comment:
To keep in line with my previous suggestions

```suggestion
    3DMMForever::AudioMan
```

Reply:
Thanks for the suggestions!